### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ionutbalutoiu**
Add `Configuration` section to the `README.md` and document the `AZDCAP_CACHE` environment variable.

Fixes https://github.com/Microsoft/Azure-DCAP-Client/issues/43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ See [src/Linux/README.MD](src/Linux/README.MD).
 
 See [src/Windows/README.MD](src/Windows/README.MD).
 
+# Configuration
+
+The Azure-DCAP-Client library uses the following environment variables if set:
+
+* `AZDCAP_CACHE` - Represents the base directory where the library cache directory `.az-dcap-client` is created. The default value is `$HOME`.
+
```
Comment (msft-jonnadul) [inline README.md]: Can you also add a snippet about how you can configure the base url for the pck cache service here?

Comment (ionutbalutoiu) [reply README.md]: Hello @msft-jonnadul!

Are you referring to these recently added environment variables ?
https://github.com/Microsoft/Azure-DCAP-Client/blob/1a82c45fe1150b377a9ff2451cde5c599242321e/src/dcap_provider.cpp#L26-L32

I thought about documenting them, but then I noticed the comment from @johnkord in the code.

However, I also think it's worth documenting them. What do you think @johnkord ?

Comment (ionutbalutoiu) [reply README.md]: @msft-jonnadul I've also added some notes about these environment variables.

Please let me know if everything looks good to you now.

**Comment by ionutbalutoiu** [comment]
@msft-jonnadul Let me know if the current changes from the PR look to be merged.

**Comment by msft-jonnadul** [comment]
@ionutbalutoiu your PR looks good to me. Feel free to merge.

**Pull request merged**